Index: alpine/imap/src/osdep/unix/flocklnx.c
===================================================================
--- alpine.orig/imap/src/osdep/unix/flocklnx.c	2014-01-30 17:01:26.028211191 -0800
+++ alpine/imap/src/osdep/unix/flocklnx.c	2014-01-30 17:01:26.008211191 -0800
@@ -57,7 +57,7 @@
     case ENOLCK:		/* lock table is full */
       sprintf (tmp,"File locking failure: %s",strerror (errno));
       mm_log (tmp,WARN);	/* give the user a warning of what happened */
-      if (!logged++) syslog (LOG_ERR,tmp);
+      if (!logged++) syslog (LOG_ERR, "%s", tmp);
 				/* return failure if non-blocking lock */
       if (op & LOCK_NB) return -1;
       sleep (5);		/* slow down in case it loops */
Index: alpine/alpine/mailcmd.c
===================================================================
--- alpine.orig/alpine/mailcmd.c	2014-01-30 17:01:26.028211191 -0800
+++ alpine/alpine/mailcmd.c	2014-01-30 17:01:26.012211191 -0800
@@ -7919,7 +7919,7 @@
 
 	      case 13 :			/* Match my addresses */
 		me++;
-		snprintf(sstring, sizeof(sstring), not ? _(dont_match_me) : _(match_me));
+		snprintf(sstring, sizeof(sstring), "%s", not ? _(dont_match_me) : _(match_me));
 		continue;
 
 	      case 14 :			/* Subject: default */
Index: alpine/pith/ldap.c
===================================================================
--- alpine.orig/pith/ldap.c	2014-01-30 17:01:26.028211191 -0800
+++ alpine/pith/ldap.c	2014-01-30 17:01:26.012211191 -0800
@@ -778,7 +778,7 @@
 	 */
 	switch(args){
 	  case 0:
-	    snprintf(filter, sizeof(filter), filt_format);
+	    snprintf(filter, sizeof(filter), "%s", filt_format);
 	    break;
 	  case 1:
 	    snprintf(filter, sizeof(filter), filt_format, scp);
