From e2eef589799d742ea6ccaec9144dc619a516222e Mon Sep 17 00:00:00 2001
From: Unknown <unknown@sourceforge.net>
Date: Sun, 3 Jun 2012 11:59:09 +0200
Subject: [PATCH] Update to RFC 5751 as per bug 3428168

---
 pith/smime.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

Index: alpine/pith/smime.c
===================================================================
--- alpine.orig/pith/smime.c	2014-01-30 17:01:36.868211247 -0800
+++ alpine/pith/smime.c	2014-01-30 17:01:36.856211247 -0800
@@ -1390,7 +1390,9 @@
     dprint((9, "encrypt_outgoing_message()"));
     smime_init();
 
-    cipher = EVP_des_cbc();
+//    cipher = EVP_des_cbc();
+// AJB 20111025 - upgrade to 128 bit AES as per RFC5751
+    cipher = EVP_aes_128_cbc();
 
     encerts = sk_X509_new_null();
 
